lib/main.dart

```diff
@@ -36,8 +36,7 @@ class _RandomWordsState extends State<RandomWords> {
     return ListView.builder(
         padding: const EdgeInsets.all(16.0),
         itemBuilder: /*1*/ (context, i) {
-          if (i.isOdd) return const Divider();
-          /*2*/
+          if (i.isOdd) return const Divider(); /*2*/
```

Comment:
Those comments with numbers (/1/, /2/, /3/, /4/) are insufficient. Consider changing them so that they carry more information, or simply remove them as your code is self-explanatory :)